components/RouteGuard.tsx

```diff
@@ -41,10 +51,15 @@ export default function RouteGuard({ children }: { children: ReactNode }) {
     }
   }
 
+  // prevent rendering until authorized is set
+  if (authorized === null) {
```

Comment:
please use double == on null checks (to cover undefined cases)

Reply:
actually. do we need this null check at all? we have `{!authorized && <></>}`  right underneath

Reply:
Removed `if statement`. Phillip brought up that the JSX is doing the same as the if statement especially with the loose equality change.